### PR TITLE
Adding mmdetection instance segmentation models

### PR DESCRIPTION
**Pull request by nvijayrania**
Todos:
- [x] @vadthyavath To add the visualization images

### CODE REVIEW

File: assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/description.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+In this paper, we aim to design an efficient real-time object detector that exceeds the YOLO series and is easily extensible for many object recognition tasks such as instance segmentation and rotated object detection. To obtain a more efficient model architecture, we explore an architecture that has compatible capacities in the backbone and neck, constructed by a basic building block that consists of large-kernel depth-wise convolutions. We further introduce soft labels when calculating matching costs in the dynamic label assignment to improve accuracy. Together with better training techniques, the resulting object detector, named RTMDet, achieves 52.8% AP on COCO with 300+ FPS on an NVIDIA 3090 GPU, outperforming the current mainstream industrial detectors. RTMDet achieves the best parameter-accuracy trade-off with tiny/small/medium/large/extra-large model sizes for various application scenarios, and obtains new state-of-the-art performance on real-time instance segmentation and rotated object detection. We hope the experimental results can provide new insights into designing versatile real-time object detectors for many object recognition tasks.
```
Comment (rjaincc) [inline assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/description.md:1]: @nvijayrania - Can you please align description to the new format?

Comment (nvijayrania) [reply assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/description.md:2]: Updated.

**Review by vadthyavath — APPROVED** [review]

File: assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mmd-3x-rtmdet-ins_x_8xb16-300e_coco
+path: ./
+properties:
+  SharedComputeCapacityEnabled: true
+  SHA: 4d77feb7a8e5967686d9bf922d2bb61162de57d2
```
Comment (rjaincc) [inline assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/spec.yaml:6]: I thought we are not keeping SHA anymore, are we keeping it?

**Review by rjaincc — APPROVED** [review]
Approving with a minor comment

File: assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/spec.yaml (hunk 2 above)
Comment (nvijayrania) [reply assets/models/system/mmd-3x-rtmdet-ins_x_8xb16-300e_coco/spec.yaml:6]: Just discussed it with @vadthyavath. We are keeping it as of now as per the design doc. 
https://microsoftapc-my.sharepoint.com/:w:/r/personal/ayushmishra_microsoft_com/Documents/System%20Registry%20model%20card%20alignment.docx?d=w2e13db38a1a94148b1dcac14a76e1f72&csf=1&web=1&e=4kS3VU

**Pull request merged**